infra/dev/docker-compose.yml

```diff
@@ -19,6 +24,8 @@ services:
     container_name: dev_laravel
     volumes:
       - ../../src:/var/www
+      - node_modules:/var/www/node_modules/
+      - vendor:/var/www/vendor/
```

Comment:
This only works one way Host->Docker
But the sync from Docker->Host still working

Reply:
As per [documentation](https://docs.docker.com/storage/volumes) it's not possible to have a subfolder of host be excluded from sync. Nearest option is RO from Docker, or tmpfs but changes are lost at each reboot.

Reply:
See [ŧhis](https://stackoverflow.com/a/55962619/2828700) also